collections/vegetation-indices.yaml

```diff
@@ -142,8 +142,8 @@ Extent:
   temporal:
     interval:
       -
-        - '2016-10-01T00:00:00Z'
-        - '2021-03-01T00:00:00Z'
+        - '2016-10-01T09:20:22.026Z'
```

Comment:
I know this is how the extent is returned by the services but maybe stick to the format HH:MM:SS (as it's in the [STAC specs)](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#temporal-extent-object). For the start date let's round this down to `02` and for the end time up to `03`. Same below

Reply:
> For the start date let's round this down to `02` and for the end time up to `03`. 

Since the returned time is displayed as HH:MM:SS.SSS, should I just remove the milisecond (SSS) part to keep the HH:MM:SS format in this case?

Reply:
Sorry, I didn't see the milliseconds part before. I suggest to remove it, but on the upper interval add 1 second so we have for sure all data included.